Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:2-onbuild
```

Comment:
is this python2 or python3? Ensure is python3. Python 2 is obsolete, we keep it only for backward compatibility with dinosaurs ;-)